packages/shared/src/components/post/SquadPostContent.tsx

```diff
@@ -67,6 +74,10 @@ function SquadPostContent({
     return shoudShowSummary ? height : 0;
   }, [shoudShowSummary, height]);
 
+  useEffect(() => {
+    sendViewPost(post.id);
```

Comment:
Isn't it better to use react query mutation here? This is more aligned with they way we do it usually